doc/devel/documenting_mpl.rst

```diff
@@ -30,11 +61,18 @@ or::
 
   ./make.py html
 
-you can also pass a ``latex`` flag to make.py to build a pdf, or pass no
-arguments to build everything.
+There are many other flags you can pass to ``make.py``, and you can see the
+full list inside that file. Here are two useful ones:
+
+* ``clean`` will delete the built sphinx files. Use this command if you're getting strange
+  errors about missing paths or broken links, particularly if you move files around.
+* ``latex`` builds a pdf of the documentation.
+
+In addition, these are useful flags:
 
-The output produced by Sphinx can be configured by editing the :file:`conf.py`
-file located in the :file:`doc/`.
+* ``--allowsphinxwarnings`` will allow the docs to continue building even if sphinx
```

Comment:
omg my life changed when I found out about this. Instead of re-running for each warning, one at a time, you can just see the output of all the warnings, change them all in one swoop, and then re-build when you want to!